code/modules/mob/living/carbon/human/human_defines_vr.dm

```diff
@@ -11,6 +11,8 @@
 	var/sensorpref = 5		//Suit sensor loadout pref
 	var/wings_hidden = FALSE
 
+	var/unnaturally_resized = FALSE 		//If got bigger than 200% or smaller than 25%
```

Comment:
Adding a variable to carbon/human just for this seems a bit ludicrous. 

Also this comment's grammar is rather sketchy

Reply:
This variable is needed if, for some reason, we want someone walk around very big or very small outside of designated areas. So in case, for example, admins resize target, this flag will stay FALSE, and game won't auto-resize you back.